### PR TITLE
setup react

### PR DESCRIPTION
**Pull request by Ntare22**
setup github actions
setup linters

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        key={book.id}
+        genre={book.genre}
+        title={book.title}
+        author={book.author}
+      />
+    ))}
+    <AddBook />
+  </div>
+);
+
```
Comment (bimbolabuari) [inline src/components/Books.js:39]: As required, Lets add a remove button 👍 
> Books - the default view
>Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet)

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @Ntare22 , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Add React-Redux (npm install react-redux).
- Add React Router and set two <Route>s and <Link>s for the app's navigation

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/components/Books.js (hunk 1 above)
Comment (Ntare22) [reply src/components/Books.js:39]: Hi @bimbolabuari,

The remove button was added.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Ntare22,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**